### ISSUE TITLE
Warning when using boolean type expression in attribute binding 

### ISSUE THREAD

**Issue by runem** [runem/lit-analyzer#15]
An attribute binding using a boolean type expression will result in the following problem.

```javascript
html`<input disabled="${true}" />`  /* <input disabled="true" /> */
html`<input disabled="${false}" />`  /* <input disabled="false" /> */
```

Both examples would result in a `disabled` input field! Using "?" would be the correct way of doing it.
```javascript
html`<input ?disabled="${true}" />`  /* <input disabled /> */
html`<input ?disabled="${false}" />`  /* <input /> */
```

Therefore this plugin should warn when assigning a boolean to an attribute and should suggest using "?" instead.

**Comment by runem** [comment]
https://github.com/runem/ts-lit-plugin/commit/c883d5d6390637b981ee2358c3f4818b5bd05ae2

**Comment by runem** [comment]
Live in version [1.1.6](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin)
